scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
```

Comment:
:stop_sign: **Security Vulnerability**: This function is vulnerable to command injection attacks[^2]. User input should never be directly used in command execution without proper sanitization.
```suggestion
import { exec } from 'child_process'
import { promisify } from 'util'

const execAsync = promisify(exec)

async function executeCommand(userInput: string) {
    try {
        // Use a whitelist of allowed commands or strictly validate input
        if (!/^[a-zA-Z0-9\s-]+$/.test(userInput)) {
            throw new Error('Invalid input')
        }
        const { stdout, stderr } = await execAsync(`ls ${userInput}`)
        console.log('stdout:', stdout)
        console.error('stderr:', stderr)
    } catch (error) {
        console.error('Error executing command:', error)
    }
}
```